### PR TITLE
Update ds-chat CI workflow paths to include zero stage 1-3 files

### PR DESCRIPTION
**Pull request by lekurile**
This PR updates the ds-chat CI workflow to run when ZeRO stage 1-3 files are updated.

### CODE REVIEW

**Review by loadams — APPROVED** [review]
Should we wait until we have the workflow passing again to merge this?

**Comment by lekurile** [comment]
> Should we wait until we have the workflow passing again to merge this?

Yes, I think we should wait until the revert PR goes in. I've done a manual test of that PR, so it will work one that goes in. Could you please review https://github.com/microsoft/DeepSpeed/pull/5461?

**Pull request merged**
